README.md

```diff
@@ -7,3 +7,93 @@ Tool for converting specialized annotated Verilog models into XML needed for
 
 Documentation can be found at https://python-symbiflow-v2x.readthedocs.io/en/latest/
 
+## Installation
+v2x can be installed from a local git repository using pip.
+
+```
+cd python-symbiflow-v2x
+pip install .
+```
+
+Alternatively, it can be installed from GitHub directly.
+
+```
+pip install git+https://github.com/SymbiFlow/python-symbiflow-v2x.git
+```
+
+## Usage
+After installing v2x, you can run `python -m v2x` to use it.
```

Comment:
We should make sure that a `v2x` binary is installed using [`setup.py`'s script entry points](https://python-packaging.readthedocs.io/en/latest/command-line-scripts.html).